automation/tji_emailer.py

```diff
@@ -0,0 +1,72 @@
+import boto3
+from botocore.exceptions import ClientError
+
+# Replace sender@example.com with your "From" address.
+# This address must be verified with Amazon SES.
+SENDER = "Aiden Yang <aiden.yang@texasjusticeinitiative.org>"
```

Comment:
Shouldn't this (and RECIPIENTS and AWS_REGION) be environment variables too?

Reply:
I'll pull all these out to a .yaml, along with some configs for the sheet checker